omnibus/package-scripts/chef/postinst

```diff
@@ -99,10 +99,22 @@ ln -sf $INSTALLER_DIR/bin/chef-client $PREFIX/bin || error_exit "Cannot link che
 chown -Rh 0:0 $INSTALLER_DIR
 
 # make the base structure for chef to run
-mkdir -p /etc/chef/client.d/
-mkdir -p /etc/chef/accepted_licenses
-mkdir -p /etc/chef/trusted_certs
-mkdir -p /etc/chef/ohai/plugins
+# the sample client.rb is only written out of no chef config dir exists yet
+if ! [ -d $CONFIG_DIR ]; then
+   mkdir -p $CONFIG_DIR
+   echo -e "# The client.rb file specifies how Chef Infra Client is configured on a node" > "$CONFIG_DIR/client.rb"
+   echo -e "# See https://docs.chef.io/config_rb_client/ for detailed configuration options" >> "$CONFIG_DIR/client.rb"
+   echo -e "#" >> "$CONFIG_DIR/client.rb"
+   echo -e "# Minimal example configuration:"  >> "$CONFIG_DIR/client.rb"
+   echo -e "# node_name  \"THIS_NODE_NAME\""  >> "$CONFIG_DIR/client.rb"
+   echo -e "# chef_server_url  \"https://CHEF.MYCOMPANY.COM/organizations/MY_CHEF_ORG\""  >> "$CONFIG_DIR/client.rb"
+   echo -e "# chef_license  \"accept\""  >> "$CONFIG_DIR/client.rb"
```

Comment:
Rather than fork a million times and make this hard to read, might I instead suggest:
```shell
cat >"$CONFIG_DIR/client.rb" <<EOF
# The client.rb file specifies how Chef Infra Client is configured on a node
# See https://docs.chef.io/config_rb_client/ for detailed configuration options
#
# Minimal example configuration:
...
EOF
```

Reply:
Is that compat with sh on AIX/Solaris? According to the comments on this file I'm not old enough to edit it. @lamont-granquist?

Reply:
yeah that's fine heredocs are older than i am

Reply:
HEREDOC is done